lib/translate/BUILD

```diff
@@ -1,7 +1,24 @@
+cc_library(
+  name = "inquiry_lib",
+  srcs = ["inquiry.cc", "common.cc"],
+  hdrs = ["inquiry.h", "common.h"],
+  deps = [
+      "//third_party/spdk_defs:nvme_defs_lib",
+      "//lib:scsi_defs_lib",
+      "@com_google_absl//absl/base",
+      "@com_google_absl//absl/types:span",
+    "@com_google_absl//absl/strings",
```

Comment:
nit: spacing. unfortunately clang-format doesn't seem to do anything for BUILD files

Reply:
i don't think this was fixed lol